Products/Zuul/routers/devicemanagement.py

```diff
@@ -41,6 +43,7 @@ def deleteMaintWindow(self, uid, id):
         """
         facade = self._getFacade()
         data = facade.deleteMaintWindow(uid, id)
+        audit('UI.MaintenanceWindow.Delete', id)
```

Comment:
maintenance window IDs are not globally unique.  you should also log the uid or parent so we know which MW of this name is being deleted

Reply:
good point, i'll add that.